tee-worker/cli/lit_ts_test.sh

```diff
@@ -18,6 +18,10 @@ function usage() {
 [ $# -ne 1 ] && (usage; exit 1)
 TEST=$1
 
+cd /client-api
+pnpm install
+pnpm --filter integration-tests run $TEST:staging
```

Comment:
Maybe a silly question: do we even have the `integration-tests` package under this folder?

Reply:
Thanks! Fixed!